schema/patient.js

```diff
@@ -0,0 +1,70 @@
+const patientSchema = `
+  enum triageLevel {
+    GREEN
+    YELLOW
+    RED
+    BLACK
+    WHITE
+  }
+
+  enum status {
+    ON_SITE
+    RELEASED
+    TRANSPORTED
+  }
+
+  extend type Query {
+    patient(id: ID!): Patient
+    patients: [Patient]
+  }
+
+  extend type Mutation {
+    addPatient(
+      gender: String,
+      age: Int,
+      runNumber: Int,
+      barcodeValue: Int,
+      collectionPointId: ID!,
```

Comment:
`collectionPointId` -> `collectionPoint`
```suggestion
      collectionPoint: ID!,
```